feature/profile/src/commonMain/kotlin/com/tunjid/heron/profile/ProfileScreen.kt

```diff
@@ -520,6 +536,45 @@ private fun ProfileHeader(
     }
 }
 
+@Composable
+private fun ProfileBio(
+    description: String?,
+    onLinkTargetClicked: (LinkTarget) -> Unit,
+) {
+    val bio = description.orEmpty()
+    val textLinks = AnnotatedString(bio).links()
+
+    val annotatedText = rememberFormattedTextPost(
+        text = bio,
+        textLinks = textLinks,
+        onLinkTargetClicked = onLinkTargetClicked,
+    )
+
+    var textLayoutResult by remember { mutableStateOf<TextLayoutResult?>(null) }
+
+    BasicText(
```

Comment:
Do we need to do custom click handling for the text? 

`rememberFormattedTextPost` already returns an `AnnotatedString` with clickable spans. We should just be able to use regular Material3 `Text` here, and the click handling should work out of the box.